src/main/java/hlf/java/rest/client/model/MultiDataTransactionPayload.java

```diff
@@ -0,0 +1,12 @@
+package hlf.java.rest.client.model;
+
+import java.io.Serializable;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class MultiDataTransactionPayload extends ValidatedDataModel implements Serializable {
+  private List<String> peerNames;
```

Comment:
Are we making peerNames mandatory? In other api's this is optional

Reply:
It remains optional.